.circleci/config.yml

```diff
@@ -701,10 +752,68 @@ jobs:
                 pattern: "^add-changelog.*"
                 value: << pipeline.git.branch >>
           steps:
-            - read-workspace
+            - checkout
+            - restore-gradle-cache
             - unit-tests-androidauto
       - run: exit 0
 
+  prepare-static-analysis:
+    executor: ndk-r22-latest-executor
+    steps:
+      - checkout
+      - restore-gradle-cache
+      - write-workspace
+
+  verify-codestyle:
+    executor: ndk-r22-latest-executor
+    steps:
+      - read-workspace
+      - verify-codestyle
+
+  verify-license:
+    executor: ndk-r22-latest-executor
+    steps:
+      - read-workspace
+      - verify-license
+
+  verify-common-sdk-version:
+    executor: ndk-r22-latest-executor
+    steps:
+      - read-workspace
+      - verify-common-sdk-version
+
+  check-api-core:
+    executor: ndk-r22-latest-executor
+    steps:
+      - read-workspace
+      - check-api-core
+
+  check-api-ui:
+    executor: ndk-r22-latest-executor
+    resource_class: medium+
+    steps:
+      - read-workspace
+      - check-api-ui
```

Comment:
Is this supposed to be like this? Aren't you recursively referencing the same job?

Reply:
No, it's okay. CircleCI can differ them